package.json

```diff
@@ -22,10 +22,14 @@
         "devextreme": "19.1-next",
         "devextreme-angular": "19.1-next",
         "devextreme-themebuilder": "19.1-next",
+        "file-saver": "^2.0.2",
+        "jszip": "^3.2.2",
+        "jszip-utils": "^0.1.0",
         "less": "^3.8.1",
         "rxjs": "^6.0.0",
         "sass.js": "^0.10.11",
         "smoothscroll-polyfill": "^0.4.3",
+        "stream": "0.0.2",
```

Comment:
I do not find the usings of this library. Do we actually need it?

Reply:
Yes. Stream needed for JSZip

Reply:
If `stream` is needed for `jszip`, `jszip` should have it in dependencies. If we include this library in our package.json, I want to see `require('stream')`. We should not include extra libraries without the need and resolve issues like 'Can't resolve stream' by adding `stream` to our app. I have googled for you https://github.com/Stuk/jszip/issues/345 and https://github.com/DevExpress/devextreme-angular/issues/776#issuecomment-386706260 (everyone knows and loves GoshaFighten, especially @BingoRUS )